docs/src/config-file.md

```diff
@@ -161,4 +161,13 @@ mbtiles:
   sources:
     # named source matching source name to a single file
     mb-src1: /path/to/mbtiles1.mbtiles
+
+# Sprite configuration
+sprites:
+  paths:
+    # all SVG files in this dir will be published as a "my_images" sprite source
+    - /path/to/my_images
+  sources:
+    # SVG images in this directory will be published as a "my_sprites" sprite source
+    my_sprites: /path/to/some_dir
```

Comment:
It's minor, but this feels unnecessarily confusing to have two separate ways to configure it. I'd rather have `sources` with explicit names IMO.

Reply:
It's the same config system as for mbtiles and pmtiles. I've would rather keep them consistent. Besides, you can specify them as a simple list or even a single value instead of an object